### PR TITLE
Create eventlist-layout　【BackLog-No.6】

### PR DESCRIPTION
**Pull request by kokioba95**
PR Description Template
## Description / Purpose (checkpoints)
Created event_list-layout. Please check if there is no collapse of the layout

Fixes issue: < https://docs.google.com/spreadsheets/d/1Ear__g58irVlaI_tpzWUkf__JRcBz8u2d2DdRYKMiDk/edit?usp=sharing > <No.6>

## How To Test This PR
 1. Go event_list (Ex. http://localhost:8086/event_list )
 
 **Commands**
None

 **Screenshots**
< https://drive.google.com/open?id=1UX_at7VJu0OeH5Mb7Eo7ULK3hSw7Ws-W >

### CODE REVIEW

**Comment by hayato0309** [comment]
1. In header, there are no "Sign up" and "login" link.
2. In header, there are no "Clou-vo" on the upper left part.
https://gyazo.com/792ecbd4a04d73f4501892635fdf3d92

**Comment by kokioba95** [comment]
I modify the file and push it.
Please check this.
"modify header"

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <?php
-
 namespace App\Http\Controllers;
-
 use Illuminate\Http\Request;
-
+use App\User;
+use App\Blog;
```
Comment (AubreyV) [inline app/Http/Controllers/HomeController.php]: Please remove this, this is not needed here.

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+use Illuminate\Support\Facades\Auth;
+use Carbon\Carbon;
```
Comment (AubreyV) [inline app/Http/Controllers/HomeController.php]: This one also

File: app/Http/Controllers/HomeController.php [hunk 3]
```diff
@@ -15,14 +16,25 @@ public function __construct()
...
-    
     /**
      * Show the application dashboard.
      *
      * @return \Illuminate\Contracts\Support\Renderable
      */
     public function index()
     {
-        return view('home');
+        $blogs = Auth::user()->blogs()->orderBy('created_at', 'INC')->get();
```
Comment (AubreyV) [inline app/Http/Controllers/HomeController.php]: We do not have blogs in your application, so please remove also.

File: app/Http/Controllers/HomeController.php [hunk 4]
```diff
@@ -15,14 +16,25 @@ public function __construct()
...
+        return view('home', compact('blogs'));
+    }
+
+    public function post(Request $request)
+    {
+        // Auth::user()->blogs()->create()
+        $blog = new Blog;
```
Comment (AubreyV) [inline app/Http/Controllers/HomeController.php]: this part also

File: app/User.php [hunk 5]
```diff
@@ -36,4 +30,21 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+    public function blogs()
+    {
+        return $this->hasMany('App\Blog');
+    }
+    public function follower()
+    {
+        return $this->belongsToMany('App\User', 'relationships', 'followed_id', 'follower_id');
```
Comment (AubreyV) [inline app/User.php]: also this

File: app/User.php [hunk 6]
```diff
@@ -36,4 +30,21 @@ class User extends Authenticatable
...
+    }
+    public function following()
+    {
+        return $this->belongsToMany('App\User', 'relationships', 'follower_id', 'followed_id');
+    }
+    public function like_user()
```
Comment (AubreyV) [inline app/User.php]: and this

File: app/User.php [hunk 7]
```diff
@@ -36,4 +30,21 @@ class User extends Authenticatable
...
     ];
+    public function blogs()
+    {
+        return $this->hasMany('App\Blog');
+    }
+    public function follower()
+    {
+        return $this->belongsToMany('App\User', 'relationships', 'followed_id', 'follower_id');
+    }
+    public function following()
```
Comment (AubreyV) [inline app/User.php]: and this

File: app/User.php [hunk 8]
```diff
@@ -36,4 +30,21 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+    public function blogs()
```
Comment (AubreyV) [inline app/User.php]: and here also

File: app/User.php [hunk 9]
```diff
@@ -36,4 +30,21 @@ class User extends Authenticatable
...
+    {
+        return $this->hasMany('App\Blog');
+    }
+    public function follower()
```
Comment (AubreyV) [inline app/User.php]: this also

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 10]
```diff
@@ -1,9 +1,7 @@
 <?php
-
```
Comment (AubreyV) [inline database/migrations/2014_10_12_000000_create_users_table.php]: do not remove the spaces here, it's purpose is for readability.

File: resources/views/auth/register.blade.php [hunk 11]
```diff
@@ -55,10 +55,9 @@
                                 @endif
                             </div>
                         </div>
-
+                        <!--
                         <div class="form-group row {{ $errors->has('birth') || $errors->has('birth_year') || $errors->has('birth_month') || $errors->has('birth_day') ? ' has-error' : '' }}">
-                            <label for="birth_year" class="col-md-4 col-form-label text-md-right">Birthday　DD/MM/YYYY </label>
-
+                            <label for="birth_year" class="col-md-4 col-form-label text-md-right">Birthday</label>
                             <div class="form-group row col-md-8">
```
Comment (AubreyV) [inline resources/views/auth/register.blade.php]: this too

File: resources/views/auth/register.blade.php [hunk 12]
```diff
@@ -115,8 +114,8 @@
                                 </span>
                             @endif
                             </div>
-
                         </div>
+                        -->
```
Comment (AubreyV) [inline resources/views/auth/register.blade.php]: this also

File: resources/views/auth/register.blade.php [hunk 13]
```diff
@@ -55,10 +55,9 @@
                                 @endif
                             </div>
                         </div>
-
+                        <!--
```
Comment (AubreyV) [inline resources/views/auth/register.blade.php]: remove if you do not need

File: resources/views/event_list.blade.php [hunk 14]
```diff
@@ -0,0 +1,126 @@
+<!doctype html>
```
Comment (AubreyV) [inline resources/views/event_list.blade.php]: I think you should extend from your master file. Review our discussion about the Blade Directives, or read here from the documentation: https://laravel.com/docs/5.8/blade#extending-a-layout

File: app/Http/Controllers/HomeController.php (hunk 2 above)
Comment (kokioba95) [reply app/Http/Controllers/HomeController.php]: It has been deleted.

File: app/Http/Controllers/HomeController.php (hunk 1 above)
Comment (kokioba95) [reply app/Http/Controllers/HomeController.php]: It has been deleted.

File: app/Http/Controllers/HomeController.php (hunk 3 above)
Comment (kokioba95) [reply app/Http/Controllers/HomeController.php]: It has been deleted.

File: app/Http/Controllers/HomeController.php (hunk 4 above)
Comment (kokioba95) [reply app/Http/Controllers/HomeController.php]: It has been deleted.

File: app/User.php (hunk 5 above)
Comment (kokioba95) [reply app/User.php]: It has been deleted.

File: app/User.php (hunk 6 above)
Comment (kokioba95) [reply app/User.php]: It has been deleted.

File: app/User.php (hunk 7 above)
Comment (kokioba95) [reply app/User.php]: It has been deleted.

File: app/User.php (hunk 8 above)
Comment (kokioba95) [reply app/User.php]: It has been deleted.

File: app/User.php (hunk 9 above)
Comment (kokioba95) [reply app/User.php]: It has been deleted.

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 10 above)
Comment (kokioba95) [reply database/migrations/2014_10_12_000000_create_users_table.php]: I've fixed it back.

File: resources/views/auth/register.blade.php (hunk 11 above)
Comment (kokioba95) [reply resources/views/auth/register.blade.php]: Changed in line with the latest Wire Frame.

File: resources/views/auth/register.blade.php (hunk 12 above)
Comment (kokioba95) [reply resources/views/auth/register.blade.php]: Changed in line with the latest Wire Frame.

File: resources/views/auth/register.blade.php (hunk 13 above)
Comment (kokioba95) [reply resources/views/auth/register.blade.php]: Changed in line with the latest Wire Frame.

File: resources/views/event_list.blade.php (hunk 14 above)
Comment (kokioba95) [reply resources/views/event_list.blade.php]: I checked the document.
I used Blade Templates.

File: app/User.php [hunk 15]
```diff
@@ -1,24 +1,19 @@
 <?php
-
```
Comment (AubreyV) [inline app/User.php]: Again, please refrain from removing these spaces because these spaces are making the code readable.

File: app/User.php [hunk 16]
```diff
@@ -1,24 +1,19 @@
...
 namespace App;
-
 use Illuminate\Notifications\Notifiable;
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Foundation\Auth\User as Authenticatable;
-
```
Comment (AubreyV) [inline app/User.php]: do not remove the space here

File: app/User.php [hunk 17]
```diff
@@ -1,24 +1,19 @@
...
 class User extends Authenticatable
 {
     use Notifiable;
-
```
Comment (AubreyV) [inline app/User.php]: here also

File: app/User.php [hunk 18]
```diff
@@ -1,24 +1,19 @@
...
     /**
      * The attributes that are mass assignable.
      *
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'first_name', 'last_name', 'group_name', 'email', 'password',
     ];
-
```
Comment (AubreyV) [inline app/User.php]: and here

File: app/User.php [hunk 19]
```diff
@@ -27,7 +22,6 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
-
```
Comment (AubreyV) [inline app/User.php]: and here

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 20]
```diff
@@ -3,7 +3,6 @@
 use Illuminate\Support\Facades\Schema;
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
-
```
Comment (AubreyV) [inline database/migrations/2014_10_12_000000_create_users_table.php]: and this

File: app/User.php (hunk 15 above)
Comment (kokioba95) [reply app/User.php]: I've fixed it back.

File: app/User.php (hunk 16 above)
Comment (kokioba95) [reply app/User.php]: I've fixed it back.

File: app/User.php (hunk 17 above)
Comment (kokioba95) [reply app/User.php]: I've fixed it back.

File: app/User.php (hunk 18 above)
Comment (kokioba95) [reply app/User.php]: i make space.

File: app/User.php (hunk 19 above)
Comment (kokioba95) [reply app/User.php]: i make space.

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 20 above)
Comment (kokioba95) [reply database/migrations/2014_10_12_000000_create_users_table.php]: i make space.

File: app/User.php [hunk 21]
```diff
@@ -1,11 +1,14 @@
 <?php
+
 namespace App;
 use Illuminate\Notifications\Notifiable;
 use Illuminate\Contracts\Auth\MustVerifyEmail;
 use Illuminate\Foundation\Auth\User as Authenticatable;
 class User extends Authenticatable
+
```
Comment (AubreyV) [inline app/User.php]: There should be no space here. it should be before the class

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 22]
```diff
@@ -4,6 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 class CreateUsersTable extends Migration
+
```
Comment (AubreyV) [inline database/migrations/2014_10_12_000000_create_users_table.php]: There should be no space here, it should be before the class.

File: app/User.php (hunk 21 above)
Comment (kokioba95) [reply app/User.php]: I made a mistake.
The code has been fixed.

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 22 above)
Comment (kokioba95) [reply database/migrations/2014_10_12_000000_create_users_table.php]: I made a mistake.
The code has been fixed.

**Pull request merged**